index.md

```diff
@@ -13,9 +13,14 @@ Fabric is a lightweight, experimental modding toolchain for Minecraft.
 
 To install Fabric, you need the **Fabric Loader**, which you can <a class="page-link page-link-download" href="/use">download here</a>
 
-In addition, most mods depend on an API mod, containing commonly used hooks. They can be found here:
+You should also read the [Fabric Wiki installation instructions](https://fabricmc.net/wiki/install)!
```

Comment:
Doesnt read quite right to me (Although I could be wrong), maybe:

```suggestion
You can also read the [installation instructions on the Fabric Wiki](https://fabricmc.net/wiki/install)!
```

Reply:
Personally I feel like users _should_ read it, as a recommendation, but I respect that you've made a decision! :)